src/utils/constants.js

```diff
@@ -340,6 +340,11 @@ export const DEFAULT_OPTIONS_ROBOTMODEL = {};
 
 export const DEFAULT_OPTIONS_TF = {};
 
+export const DEFAULT_OPTIONS_MARKER_TEXT_VIEW_FACING  = {
```

Comment:
Right now, the options for each visualization are those that are configurable from leftbar in Zethus.
Eg: Color and alpha for range visualization.
The size and color in this case are actually primitive options and not marker options. They're similar to default radius and color for cone primitive. Marker message has scale, color which can be applied to override these but they won't be taken from user.
http://docs.ros.org/api/visualization_msgs/html/msg/Marker.html